spark-jobs/src/main/scala/filodb/downsampler/index/DSIndexJob.scala

```diff
@@ -55,25 +58,36 @@ object DSIndexJob extends StrictLogging with Instance {
   def updateDSPartKeyIndex(shard: Int, epochHour: Long): Unit = {
     import DSIndexJobSettings._
 
+    taskCounter.increment
+
+    val span = Kamon.buildSpan("timetaken-index-migration")
+      .withTag("shard", shard)
+      .withTag("epochHour", epochHour)
```

Comment:
epochHour becomes a high-card tag. Lets drop.
Also querying will result in too many time series

Reply:
removed epochHour tag